tweeter/src/main/java/Tweeter.java

```diff
@@ -0,0 +1,52 @@
+import twitter4j.*;
+
+//import java.io.IOException;
+import java.util.List;
+
+public class Tweeter {
+
+    private static Twitter tweeter = TwitterFactory.getSingleton();
```

Comment:
Change static access, unless necessary do not use static in java - https://dzone.com/articles/why-static-bad-and-how-avoid

Reply:
Thanks a lot for the link.